app/scripts/controllers/create/createFromImage.js

```diff
@@ -1,29 +1,30 @@
 "use strict";
 
 angular.module("openshiftConsole")
-  .controller("CreateFromImageController", function ($scope,
-      Logger,
-      $q,
-      $routeParams,
-      APIService,
-      DataService,
-      ProjectsService,
-      Navigate,
-      ApplicationGenerator,
-      LimitRangesService,
-      MetricsService,
-      HPAService,
-      QuotaService,
-      SecretsService,
-      ImagesService,
-      TaskList,
-      failureObjectNameFilter,
-      $filter,
-      $parse,
-      $uibModal,
-      SOURCE_URL_PATTERN,
-      keyValueEditorUtils
-    ){
+  .controller("CreateFromImageController",
+              function($scope,
+                       $filter,
+                       $parse,
+                       $q,
+                       $routeParams,
+                       $uibModal,
+                       APIService,
+                       ApplicationGenerator,
+                       DataService,
+                       HPAService,
+                       ImagesService,
+                       LimitRangesService,
+                       Logger,
+                       MetricsService,
+                       Navigate,
+                       NotificationsService,
```

Comment:
Only change to dependencies is `NotificationsService`. But I sorted and reformatted the list.